paddle/fluid/framework/details/op_handle_base.h

```diff
@@ -27,7 +27,7 @@ namespace paddle {
 namespace framework {
 namespace details {
 
-constexpr char kLocalExecScopeName[] = "@LOCAL_SCOPE@";
+constexpr char kLocalExecScopeName[] = "@LOCAL_EXE_SCOPE@";
```

Comment:
Why change this?

Reply:
To make the name more explicitly. The local scope and local execution scope are different.